frontend/packages/data-portal/app/components/Dataset/RunsTable.tsx

```diff
@@ -61,6 +62,24 @@ export function RunsTable() {
     [searchParams, setSingleDatasetHistory],
   )
 
+  const getRunUrl = useCallback(
+    (id: number) => {
+      const url = createUrl(`/runs/${id}`)
+
+      if (deposition) {
+        maybeAddDepositionIdFilter({
```

Comment:
wonder if we could extract the filter carryover logic separately from the add deposition filter logic? Or maybe just rename this function? Kinda confused me cuz I thought all this function was doing was just tacking on 1 param